server.go

```diff
@@ -35,9 +58,70 @@ func (server *Server) Serve(param ServerParam) {
 	fmt.Printf("accepting connections at http://*:%d/\n", port)
 	http.Handle("/", contentHandler(param, http.FileServer(http.Dir("."))))
 	http.Handle("/revealjs/", assetsHandler("/assets/", http.FileServer(http.FS(revealjs))))
+
+	if param.Multiplex.Secret != "" {
+		socketioServer := setupSocketIO()
+		go func() {
+			if err := socketioServer.Serve(); err != nil {
+				log.Fatalf("socketio listen error: %s\n", err)
+			}
+		}()
+		defer socketioServer.Close()
+
+		param.Multiplex.IsMaster = true
+		http.Handle("/master/", http.StripPrefix("/master", contentHandler(param, http.FileServer(http.Dir(".")))))
+		http.Handle("/socketio/", assetsHandler("/assets/", http.FileServer(http.FS(socketiojs))))
+		http.Handle("/socket.io/", socketioServer)
```

Comment:
I was thinking that the /socketio/ and /revealjs/ handlers could be merged as assets/. But this will require to update all the paths in the slide.html, which could be a considerable change. If you are alright with that I can do it.